### DISCUSSION TITLE
Carbs are not synchronized to xDrip sync follower (AAPS 3.0)

### DISCUSSION THREAD

**Discussion by vmichalec** [Navid200/xDrip#178] [Development]
Hi, I'm using AAPS 3.0 (still beta) with xDrip that accepts treatments. On master phone it works like it should. I have also xDrip sync follower on another phone but here is the problem. Carbs are not shown here.

AAPS
![AAPS](https://user-images.githubusercontent.com/24793786/147849726-cd133c11-4b46-4e06-a3d2-11e99273a8c7.png)

xDrip sync master - here with insulin and carbs shown
![xDrip+sync-master](https://user-images.githubusercontent.com/24793786/147849734-05593a20-b528-41f1-8c29-4a5360f6e5ae.png)

xDrip sync follower - here are only insulin entries but no carbs
![xDrip+sync-follower](https://user-images.githubusercontent.com/24793786/147849742-1e85afbc-9bf1-4545-9a9b-e5b9351dab81.jpg)

Nightscout treatments - entries as seen on nightscout
![Nightscout_treatments](https://user-images.githubusercontent.com/24793786/147849769-27132895-6615-48c0-938f-6f492a48cbfb.jpg)

I have checked it on 2021.12.24 version of xDrip. (It was working with previous AAPS versions.)

**Comment by Navid200** [comment]
Thanks
Would you please be patient until I get the right person to look at this?

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.

**Reply by vmichalec** [reply]
Ok, now I see... I have used button on https://github.com/NightscoutFoundation/xDrip/issues/new/choose which navigated me to this xDrip fork. So should I copy this issue to https://github.com/NightscoutFoundation/xDrip/discussions ?

**Reply by Navid200** [reply]
I am sorry about that.
You are absolutely correct.  That link will be updated as soon as a PR I have opened will be merged, which I hope will be in the next few days.

It will be wonderful if you could copy.

@tolot27
Can you copy this discussion?
My permissions don't allow me to.

**Comment by tolot27** [comment]
> xDrip sync follower - here are only insulin entries but no carbs

@vmichalec Please can you confirm that you were talking about "xDrip+ Sync Follower" and not "Nightscout Follower"?

During my tests with "xDrip+ Sync Follower" treatments were synced but if I switch to "Nightscout Follower" they will not by synced even if "Download Treatments" is checked. For the later there is a bug report: https://github.com/NightscoutFoundation/xDrip/issues/1196

**Reply by vmichalec** [reply]
My config for master phone xDrip is:
![master_xdrip](https://user-images.githubusercontent.com/24793786/148506727-b0650e1c-b047-48d2-881d-77949d2cef3c.png)
and data provider is AAPS & BYODA:
![master_aaps](https://user-images.githubusercontent.com/24793786/148506738-c3d14372-d5ba-44ec-b637-707ebb7fd0fa.png)

and follower phone xDrip:
![follower](https://user-images.githubusercontent.com/24793786/148506764-aab2fb8c-6b38-4553-a662-1197183e4158.jpg)

But here if I enter only carbs in AAPS - those will be synced to follower, if I enter combined insulin&carbs using bolus wizard in AAPS - those are shown on master phone xDrip but are not transferred to follower.

**Comment by vmichalec** [comment]
I copied/moved this question to https://github.com/NightscoutFoundation/xDrip/discussions/1935

**Reply by Navid200** [reply]
Thanks
